components/alertbanner/stories/alertbanner.stories.js

```diff
@@ -64,36 +70,213 @@ export default {
+					})}
+				</div>
+			</div>
+		`)}
+
+		<div 
+			class="spectrum-Typography"
+			style=${styleMap(smallerThanMaxWidth)}
+		>
+			${Typography({
+				semantics: "detail",
+				size: "l",
+				content: ["Container smaller than max-width"],
+			})}
+			<div
+				style=${styleMap({
+					display: "flex",
+					flexDirection: "column",
+					gap: "12px",
+				})}
+			>
+				${Template({
+					...args,
+					variant: "info",
+					text: "Your trial will expire in 3 days. Once it expires your files will be saved and ready for you to open again once you have purchased the software.",
+				})}
+			</div>
+		</div>
+	`;
+};
+
+/**
+ * Examples of all text overflow (wrapping) scenarios.
+ */
+const TextOverflowTemplate = (args) => html`
 	<div
 		style=${styleMap({
-			"display": "flex",
-			"flex-direction": "column",
-			"gap": "16px",
+			display: "flex",
+			flexDirection: "column",
+			gap: "12px",
 		})}
 	>
 		${Template({
 			...args,
+			variant: "info",
+			text: "Your trial will expire in 3 days. Once it expires your files will be saved and ready for you to open again once you have purchased the software.",
 		})}
-		${window.isChromatic() ?
-		Template({
+		${Template({
 			...args,
-			hasActionButton: true,
-			variant: "info",
-			text: "Your trial will expire in 3 days. Once it expires your files will be saved and ready for you to open again once you have purcahsed the software."
-		}): null }
-		${window.isChromatic() ?
-				Template({
-					...args,
-			hasActionButton: true,
-			variant: "negative",
-			text: "Connection interupted. Check your network to continue."
-		})
-		: null }
+			variant: "neutral",
+			text: "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.",
+			actionButtonText: "",
+		})}
+	</div>
+`;
+
+/**
+ * Examples of options for contextual action button and close button.
+ */
+const ActionableOptionsTemplate = (args) => html`
+	<div
+		style=${styleMap({
+			display: "flex",
+			flexDirection: "column",
+			gap: "12px",
+		})}
+	>
+		${Template({
+			...args,
+			variant: "neutral",
+			text: "Your trial has expired.",
+			actionButtonText: "Buy now",
+		})}
+		${Template({
+			...args,
+			variant: "neutral",
+			text: "Your trial has expired",
+			actionButtonText: "",
+		})}
 	</div>
 `;
 
+/**
+ * Stories
+ */
 export const Default = AlertBannerGroup.bind({});
 Default.args = {};
+
+/**
+ * Stories for the MDX "Docs" only.
+ * Based off of the base `Template` which does not have conditional Chromatic-only markup.
+ */
+export const Neutral = Template.bind({});
+Neutral.tags = ["is-hidden-story"];
```

Comment:
Heads up we renamed this to `docs-only` in [another PR](https://github.com/adobe/spectrum-css/pull/2770#discussion_r1610254723) 

```suggestion
Neutral.tags = ["docs-only"];
```